### DISCUSSION TITLE
Spelling Plugin Can not Load

### DISCUSSION THREAD

**Discussion by limopc** [zim-desktop-wiki/zim-desktop-wiki#2130] [General]
Hi all,
First, I am new here, but using Zim for long time.
I am on EndeavourOS, KDE Plasma, BTRFS file system.
I believe I have installed requirements for spell checking and it spell check is working fine on many other apps.

Still, Edit -> Preferences -> Plugins Tab, "Spell" is way down at the bottom below the alphabetical order and I have a red message saying " There was a problem loading this plugin"

What might be wrong?
I will appreciate any help to fix it.
Thank you.

**Comment by introt** [comment]
Hi! What's your Zim version and where did you get it from?
